tests/MavenTests/testEIC.cpp

```diff
@@ -241,7 +241,8 @@ void TestEIC:: testgroupPeaks() {
 void TestEIC:: testeicMerge() {
     bool matchRtFlag = true;
     float compoundRTWindow = 2;
-    float compoundPPMWindow = 10;
+    MavenParameters* mavenparameters = new MavenParameters();
+    mavenparameters->compoundMassCutoffWindow->setMassCutoffAndType(10,"ppm");
```

Comment:
Add a test case for mDa as well

Reply:
Here we can add a test for mDa only when whole test is written for this.